### DISCUSSION TITLE
Does this work on El Capitan?

### DISCUSSION THREAD

**Discussion by Runt99** [nigification/Flowgorithm-Mac-OSX#7] [Q&A] [answered]
a student is wondering how far back is this app compatible for?

**Comment by jostasik** [comment, marked as answer]
Im not entirely sure if the most recent update will operate on that OSX.. However I can confirm that my [v2.30.3](https://github.com/jostasik/Flowgorithm-macOS/releases/tag/v2.30.3-release) release will run under El Capitan without any issues!

P.S. - Since i don't have access to many different machines, I gladly accept any and all test reports from users who would like to contribute to a future compatibility table :)

**Reply by Runt99** [reply]
thanks for the update! will let him know
